### PR TITLE
Feature/contactus

### PR DESCRIPTION
**Pull request by aswiniraghul**
This PR has changes for contact us form. Users can send messages to B.R.E.W.S. using this form.
Email created to receive the messages: brewslaunchcode@outlook.com

image

### CODE REVIEW

File: IT-Wizards-UI/src/components/ContactUs.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React, { useRef, useState } from 'react';
```
Comment (joshualjames) [inline IT-Wizards-UI/src/components/ContactUs.jsx]: Please remove "React,"

File: IT-Wizards-UI/src/components/ContactUs.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import emailjs from '@emailjs/browser';
+import { Container, Row, Col, Form, Button, Alert } from 'react-bootstrap';
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const ContactUs = () => {
+  const form = useRef();
+  const [message, setMessage] = React.useState(null);
```
Comment (joshualjames) [inline IT-Wizards-UI/src/components/ContactUs.jsx]: Please remove "React."

File: IT-Wizards-UI/src/components/ContactUs.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  const [error, setError] = React.useState(null);
```
Comment (joshualjames) [inline IT-Wizards-UI/src/components/ContactUs.jsx]: Please remove "React."

**Review by joshualjames — COMMENTED** [review]
Just a couple of adjustments for the react import to align with how we are importing throughout the rest of the application.

File: IT-Wizards-UI/src/components/ContactUs.jsx (hunk 3 above)
Comment (aswiniraghul) [reply IT-Wizards-UI/src/components/ContactUs.jsx]: Josh, I have made changes as per your review comments. Please review.

**Comment by aswiniraghul** [comment]
Josh, I have made changes as per your review comments. Please review.

**Comment by aswiniraghul** [comment]
image

Also, we have received the message from you as expected.

**Comment by joshualjames** [comment]
Awesome, go ahead and merge!

**Pull request merged**